platform/jobs/edxPlatformQualityPr.groovy

```diff
@@ -0,0 +1,141 @@
+package platform
+
+import org.yaml.snakeyaml.Yaml
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_LOG_ROTATOR
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_GITHUB_BASEURL
+
+/*
+Example secret YAML file used by this script
+publicJobConfig:
+    open : true/false
+    jobName : name-of-jenkins-job-to-be
+    repoName : name-of-github-edx-repo
+    platformUrl : platform-github-url-segment.git
+    platformCredential : n/a
+    platformCloneReference : clone/.git
+    protocol : https
+*/
+
+/* stdout logger */
+/* use this instead of println, because you can pass it into closures or other scripts. */
+/* TODO: Move this into JenkinsPublicConstants, as it can be shared. */
```

Comment:
We should either take the time to refactor this, or stop marking it as a TODO in all the new files. Do you know if there is a ticket in the backlog that we can estimate and discuss at sprint planning?